docs/controllers/services/annotations.md

```diff
@@ -101,6 +101,10 @@ Specifies the hostname used for the Service `status.Hostname` instead of assigni
 
 Specifies which algorithm the Load Balancer should use. Options are `round_robin`, `least_connections`. Defaults to `round_robin`.
 
+## service.beta.kubernetes.io/do-loadbalancer-size-slug
+
+Specifies which size the Load Balancer should be created with. Options are `lb-small`, `lb-medium`, and `lb-large`. Defaults to `lb-small`.
```

Comment:
If we plan to release the next CCM by the team the slug sizes will already be available backend-wise, then I think this update to the annotation docs suffices. For the case where it might still take us some time to honor the new parameter, however, I'd suggest to merge this change with a small comment saying that the feature might not be available yet.

The context of this comment here is that many DOKS customers rely on the annotation docs to reflect the current state, so we should make sure there's no confusion for customers that try it out with the next CCM release we ship but don't immediately see that the feature won't be effective yet.

Let's clarify on the timeline internally.

Reply:
Yup that makes sense; I have no issue with keeping this blocked until the time comes 👍

Reply:
Resolving this for now, I've added a note to the annotation that this feature is in closed beta.